README.md

```diff
@@ -32,9 +32,11 @@ With simple clicks, hiring managers can find potential candidates and conduct in
 | Frew Weldemariam |https://github.com/FruitsOfGlory1|  JavaScript, PHP, Java, Python, SQL  |
 
 ## Technologies Used: 
-Frontend : HTML, CSS, JAVASCRIPT, React.
```

Comment:
Currently, the app is using JSX which then translates into HTML but I think it's fine.